xbmc/guilib/GUIMoverControl.h

```diff
@@ -15,27 +15,28 @@
 
 #include "GUIControl.h"
 #include "GUITexture.h"
+#include "utils/MovingSpeed.h"
 
 #define ALLOWED_DIRECTIONS_ALL   0
 #define ALLOWED_DIRECTIONS_UPDOWN  1
 #define ALLOWED_DIRECTIONS_LEFTRIGHT 2
 
-#define DIRECTION_NONE 0
```

Comment:
Just a note, likely doesn't apply here but be careful with defines sometimes they get exposed to the python api

Reply:
i tried to make a search but i havent found no reuse of this define